_posts/patterns/2015-04-15-icons.md

```diff
@@ -547,7 +558,7 @@ info:               Icons are used to clearly and visually represent some inform
             </span>
         </div>
         <div class="icon-classname">
-            Sort
+            sort
```

Comment:
"unsorted" is listed in synonyms.  Should it be listed here too or removed from the synonyms?